.github/workflows/cd.yml

```
@@ -20,16 +20,22 @@ jobs:
     runs-on: ubuntu-latest
     environment:
       name: pypi
-      url: https://pypi.org/p/mqt.bench
+      url: https://pypi.org/p/mqt.predictor
     permissions:
       id-token: write
+      attestations: write
+      contents: read
     needs: [python-packaging]
     steps:
       - uses: actions/download-artifact@v4
         with:
           pattern: cibw-*
           path: dist
           merge-multiple: true
+      - name: Generate artifact attestation for sdist and wheel(s)
+        uses: actions/attest-build-provenance@v1.4.0
```

```suggestion
        uses: actions/attest-build-provenance@v1.4.1
```
IIRC, there was an update here lately. Doesn't hurt to cover this here directly.